lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -172,7 +174,8 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
       stopped_resources: parse_cluster_stopped_resources(crmmon),
       nodes: parse_cluster_nodes(payload, sid),
       sbd_devices: parse_sbd_devices(sbd),
-      sites: parse_hana_scale_out_sites(HanaArchitectureType.classic(), cib, sid)
+      sites: parse_hana_scale_out_sites(HanaArchitectureType.classic(), cib, sid),
+      hana_scenario: parse_hana_scenario(crmmon, ClusterType.hana_scale_out())
```

Comment:
Let's try to put the field in the same order. Above, you put it after `architecture_type`. Do the same everywhere